### PR TITLE
fix(docker): latest tag refers to the latest minor release of Camunda BPM platform

### PR DESCRIPTION
**Pull request by aabouzaid**
Related to: INFRA-1707

### CODE REVIEW

File: release.sh [hunk 1]
```diff
@@ -45,8 +45,9 @@ else
     fi
 fi
 
-git fetch origin next 
-if [ $(git rev-parse HEAD) = $(git rev-parse FETCH_HEAD) ]; then
```
Comment (lwille) [inline release.sh:49]: why was this deleted? How would we now make sure that we only push `-latest` tags for the current minor version?

Comment (aabouzaid) [reply release.sh:49]: @lwille as discussed, I've added both of the conditions and added a comment to clarify why they are needed.
Feel free to rephrase the wording of the comment.

**Review by lwille — APPROVED** [review]
Thank you, LGTM!
